include/nbl/asset/utils/ICPUVirtualTexture.h

```diff
@@ -132,8 +132,12 @@ class ICPUVirtualTexture final : public IVirtualTexture<ICPUImageView, ICPUSampl
 
         upscaled_img->setBufferAndRegions(std::move(buf), std::move(regions));
 
-        using blit_filter_t = asset::CBlitImageFilter<asset::VoidSwizzle,asset::IdentityDither/*TODO: White Noise*/,void,false,asset::CMitchellImageFilterKernel<>>;
-        blit_filter_t::state_type blit;
+        using blit_filter_t = asset::CBlitImageFilter<asset::VoidSwizzle,asset::IdentityDither/*TODO: White Noise*/,void,false, asset::CBlitUtilities<asset::CWeightFunction1D<asset::SMitchellFunction<>>>>;
+        auto convolutionKernels = blit_filter_t::blit_utils_t::getConvolutionKernels(
```

Comment:
the resampling filter should be Mitchell and the reconstruction should be Linear or Mitchell as well

Reply:
Both reconstruction and resampling are Mitchell
Resolved.